src/components/Card.tsx

```diff
@@ -108,11 +109,14 @@ const cardContainer = (
     useAppStyles ? appStyles : undefined
   )
 }
-const spinnerBox = classnames(
-  position('absolute'),
-  inset('-top-24', '-right-4', 'md:-top-28', 'md:-right-40')
-)
-
+const spinnerBox = (mobileSpinnerOnRight?: boolean) => {
+  return classnames(
+    position('absolute'),
+    mobileSpinnerOnRight
+      ? inset('-top-24', '-right-28', 'md:-top-28', 'md:-right-40')
+      : inset('-top-24', '-right-4', 'md:-top-28', 'md:-right-40')
```

Comment:
I think we can group the two lines here together for the ones with same attributes

Reply:
Yes, let's extract repetitive styles into `spinnerPosition`